cpp/include/cudf/detail/sorting.hpp

```diff
@@ -47,6 +47,28 @@ std::unique_ptr<column> sorted_order(
     rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource(),
     cudaStream_t stream = 0);
 
+/**
```

Comment:
Why not use copydoc ?

Reply:
because the word `stable` need to be added in brief.
@devavret suggest writing `@brief` and then use `@copydetails`